test/controller/user.test.js

```diff
@@ -0,0 +1,77 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../src/app';
+import userMock from '../data/user.mock';
+
+chai.use(chaiHttp);
+chai.should();
+
+const { user1, user1Login, userEmpty, user1wrongcredetianls } = userMock;
+
+describe('Login/Signup Test', () => {
```

Comment:
Can you `Login/Signup Test` to `User Test`, because your are testing the user controller. so it makes sense to name it `User Tests`.